server/routes/risk.js

```diff
@@ -12,95 +13,99 @@ module.exports = {
 
         if (!risk) {
-          return boom.badRequest('Invalid result', new Error('Missing calculate_flood_risk key'))
+          return boom.badRequest('Invalid result', new Error('Missing calculate_flood_risk key')) // this can be removed data service ticket is complete
         }
 
         /*
-         * If we get here we can be sure we have a valid result from
-         * the database and we can start to prepare our return response
-         */
+        * If we get here we can be sure we have a valid result from
+        * the ESRI database and we can start to prepare our return response
+        */
         let reservoirDryRisk = null
         let reservoirWetRisk = null
         let riverAndSeaRisk = null
 
-        if (risk.dry_reservoir_risk && risk.dry_reservoir_risk !== 'Error') {
-          reservoirDryRisk = risk.dry_reservoir_risk.map(function (item) {
+        if (riskQueryResult.dryReservoirs.length > 0) {
+          reservoirDryRisk = riskQueryResult.dryReservoirs.map(function (item) {
             return {
-              reservoirName: item.reservoir,
-              location: item.ngr,
-              riskDesignation: item.risk_designation,
-              undertaker: item.undertaker,
-              leadLocalFloodAuthority: item.llfa_name,
-              comments: item.comments
+              reservoirName: item.attributes.reservoir,
+              location: item.attributes.ngr,
+              riskDesignation: item.attributes.risk_designation,
+              undertaker: item.attributes.undertaker,
+              leadLocalFloodAuthority: item.attributes.llfa_name,
+              comments: item.attributes.comments
             }
           })
         } else {
-          reservoirDryRisk = risk.dry_reservoir_risk
+          reservoirDryRisk = riskQueryResult.dryReservoirs
         }
 
-        if (risk.wet_reservoir_risk && risk.wet_reservoir_risk !== 'Error') {
-          reservoirWetRisk = risk.wet_reservoir_risk.map(function (item) {
+        if (riskQueryResult.wetReservoirs.length > 0) {
+          reservoirWetRisk = riskQueryResult.wetReservoirs.map(function (item) {
             return {
-              reservoirName: item.reservoir,
-              location: item.ngr,
-              riskDesignation: item.risk_designation,
-              undertaker: item.undertaker,
-              leadLocalFloodAuthority: item.llfa_name,
-              comments: item.comments
+              reservoirName: item.attributes.RESERVOIR,
+              location: item.attributes.NGR,
+              riskDesignation: item.attributes.RISK_DESIGNATION,
+              undertaker: item.attributes.UNDERTAKER,
+              leadLocalFloodAuthority: item.attributes.LLFA_NAME,
+              comments: item.attributes.COMMENTS
             }
           })
         } else {
-          reservoirWetRisk = risk.wet_reservoir_risk
+          reservoirWetRisk = riskQueryResult.wetReservoirs
         }
 
         if (risk.rofrs_risk) {
```

Comment:
This is using a result from the database call to check the results from the ESRI Query call. This needs to be changed.